UnitsNet/Scripts/UnitDefinitions/ForcePerLength.json

```diff
@@ -0,0 +1,35 @@
+﻿{
+  "Name": "ForcePerLength",
+  "BaseUnit": "NewtonPerMeter",
+  "XmlDoc": "Force per length is the amount of change applied per length.",
+  "Units": [{
+      "SingularName": "NewtonPerMeter",
+      "PluralName": "NewtonsPerMeter",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "N/m" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "KilogramForcePerMeter",
```

Comment:
Kilogram is one of the exceptions to the SI rule. See `Mass` and `Force`.
If you specify `GramForcePerMeter` instead, you can use `Prefixes` to get `KilogramsForcePerMeter`, `MilligramsForcePerMeter` and so on. It's up to you to add whatever prefixes make sense in the context of `ForcePerLength`.